letter.js

```diff
@@ -0,0 +1,83 @@
+var Word = require("./word.js");
+var inquirer = require("inquirer");
+var wordsArr = ["dingus", "mongolia", "pump-up-the-jam", "pneumonoultramicroscopicsilicovolcanoconiosis"];
+var guesses = 10;
+
+console.log("Welcome to hangmanNode! To keep you guessing, there are no spaces between blanks so you can't guess how many characters are left (;")
+
+var Letter = function() {
+	this.newWord = function(num) {
+		// to grab constructor from Word
+		var newWord = new Word(num);
+		// function so pump-up-the-jam's '-' doesn't have to be guessed by user
+		var blanksFunc = function() {
+			for (var i=0; i<newWord.guessWord.length; i++) {
+				if (newWord.guessWord[i] === "-") {
+					blanksArr = newWord.blanks.split("");
+					blanksArr[i] = "-";
+					newWord.blanks = blanksArr.join("");
+				}
+			}
+		}
```

Comment:
This doesn't need to be a functional expression because you aren't passing around this function as a variable. Since it is doing some basic manipulation, just make it a pure function that sits outside of Letter and it just takes some word object and returns that object with the modified prop.